app/assets/javascripts/components/form/form_input.jsx

```diff
@@ -63,9 +63,10 @@ class FormInput extends Component {
         </div>
         <div style={StyleConstants.forms.questions.response}>
           <input
+            placeholder={question.placeholder}
             ref={'container'}
             style={this.styles.input}
-            placeholder={question.placeholder}
+            type={question.format}
```

Comment:
Passing an `input` element tells it to display format helpers - `'date'` tells the `input` to have a date picker!